actors/src/main/scala/zio/actors/ActorSystem.scala

```diff
@@ -43,8 +43,8 @@ object ActorSystem {
  */
 final class Context private[actors] (
   private val path: String,
-  private val actorSystem: ActorSystem,
-  private val childrenRef: Ref[Set[ActorRef[Throwable, Any]]]
+  private[actors] val actorSystem: ActorSystem,
```

Comment:
Can we avoid exposing all these internals?

Reply:
Sure, I can do it via getters for what is needed from them and make those args private again. Is that ok?

Reply:
Let's start with private-first, if we need to expose them, we can do it prior to release. WDYT?